experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -200,6 +200,8 @@ export const FormAudience = ({
     experiment.application === NimbusExperimentApplicationEnum.DESKTOP;
   const isLiveRollout = experiment.isRollout && getStatus(experiment).live;
   const isLocked = isLiveRollout && !getStatus(experiment).draft;
+  const isArchived =
+    experiment?.isArchived != null ? experiment.isArchived : false;
```

Comment:
I am curious why you have use `!= null`, I think without even that it would work

Reply:
isArchived can be `boolean | null`, unfortunately 😬